Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -582,16 +605,21 @@ def __init__(self):
         self.messages = []  # type: list
         self.write_buf = []  # type: list
         self.replace_strs = []  # type: list
+        self.buffering = True
         # if for some reason you don't want to auto add credentials.password to replace strings
         # set the os env COMMON_SERVER_NO_AUTO_REPLACE_STRS. Either in CommonServerUserPython, or docker env
-        if (not os.getenv('COMMON_SERVER_NO_AUTO_REPLACE_STRS') and hasattr(demisto, 'getParam')
-                and isinstance(demisto.getParam('credentials'), dict)
-                and demisto.getParam('credentials').get('password')):
-            pswrd = self.encode(demisto.getParam('credentials').get('password'))
-            to_encode = pswrd
-            if IS_PY3:
-                to_encode = pswrd.encode('utf-8', 'ignore')
-            self.add_repalce_strs(pswrd, base64.b64encode(to_encode))
+        if (not os.getenv('COMMON_SERVER_NO_AUTO_REPLACE_STRS') and hasattr(demisto, 'getParam')):
+            # add common params
+            if isinstance(demisto.getParam('credentials'), dict) and demisto.getParam('credentials').get('password'):
+                pswrd = self.encode(demisto.getParam('credentials').get('password'))
+                self.add_replace_strs(pswrd, b64_encode(pswrd))
+            sensitive_params = ('api_key', 'apikey', 'password', 'secret')
```

Comment:
i would add the following: key, token

Reply:
`access_key`, `private_key` could be also options

Reply:
I change to simply treat any param with key/secret/password/token/private as relevant